app/src/main/java/io/pumpkinz/pumpkinreader/model/News.java

```diff
@@ -1,62 +1,90 @@
 package io.pumpkinz.pumpkinreader.model;
 
-import org.parceler.Parcel;
-
-import java.io.Serializable;
-import java.util.Date;
 import java.util.List;
 
 
-@Parcel
-public class News implements Serializable {
+public abstract class News extends Item {
 
-    String title;
-    String body;
-    String submitter;
+    List<Integer> kids;
     String url;
-    Date date;
     int score;
+    String title;
+    int descendants;
```

Comment:
Difference concern? What difference concern?

Or do you mean `different` concern?